newapi/super/S_Page/super_page.py

```
@@ -884,3 +860,109 @@ def Create(self, text="", summary="", nodiff="", noask=False):
+        # data = self.post_params(params)
+        # data = data.get('query', {}).get('links', [])
+        # ---
+        data = self.post_continue(params, "query", _p_="links", p_empty=[])
+        # ---
+        # [{'ns': 14, 'title': 'تصنيف:مقالات بحاجة لشريط بوابات', 'exists': True}, {'ns': 14, 'title': 'تصنيف:مقالات بحاجة لصندوق معلومات', 'exists': False}]
+        # ---
+        self.links_data.links = data
+        # ---
+        return self.links_data.links
+
+    def get_revisions(self, rvprops=[]):
```

Ruff (0.11.9)

Do not use mutable data structures for argument defaults

Replace with `None`; initialize within function

(B006)